mrbgems/mruby-os-memsize/src/memsize.c

```diff
@@ -54,6 +54,7 @@ os_memsize_of_object(mrb_state* mrb, mrb_value obj)
       size += mrb_objspace_page_slot_size();
       if (!RSTR_EMBED_P(RSTRING(obj)) && !RSTR_SHARED_P(RSTRING(obj))) {
         size += RSTRING_CAPA(obj);
+        size ++; /* NUL terminator */
```

Comment:
I prefer not having space before `++` operator.

Reply:
I changed.